### PR TITLE
when no default locations are matched from assigned locations, pick from jurisdictions

### LINKED ISSUE #787: Unable to Log in when team assignment being made at a higher level
Reported here.
https://github.com/OpenSRP/opensrp-client-reveal/issues/1303

### PR DESCRIPTION
**Pull request by faidvd**
resolves #787

### CODE REVIEW

File: src/main/java/org/opensrp/web/controller/UserController.java [hunk 1]
```diff
@@ -6,16 +6,7 @@
...
-import java.util.Arrays;
-import java.util.Calendar;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TimeZone;
+import java.util.*;
```
Comment (Rkareko) [inline src/main/java/org/opensrp/web/controller/UserController.java:9]: revert this change. Let's avoid use of wildcard imports

**Comment by rehammuzzamil** [comment]
@Rkareko PR looks good to me. Maybe we can add a unit test to verify that the default location is only picked from Jurisdictions if it is not matched from the assigned locations.

**Comment by faidvd** [comment]
Test added.

**Review by Rkareko — APPROVED** [review]

**Comment by Rkareko** [comment]
@faidvd could you increment the version

**Review by Rkareko — APPROVED** [review]
